openapi2kong/openapi2kong.go

```diff
@@ -521,6 +521,27 @@ func getForeignKeyPlugins(
 	return &genericPlugins, &newPluginList
 }
 
+// findParameterSchema returns the Schema for given parameter name.
+// Path level parameters can be overridden at operation level, so we check operation parameters first
+// and then fall back to path.
+func findParameterSchema(
+	operationParameters []*v3.Parameter,
+	pathParameters []*v3.Parameter,
+	paramName string,
+) *openapibase.Schema {
+	for _, param := range operationParameters {
+		if param.Name == paramName {
+			return param.Schema.Schema()
```

Comment:
Are we sure that we won't get nil related panics here?

Reply:
While omitting `schema` from parameter does have param.Schema pointing to nil, this doesn't seem to be causing panic. I'll fix this to be safe.

Reply:
We aren't getting panic because param.Schema.Schema() is a method with pointer receiver, and is valid even if the receiver is nil - https://github.com/pb33f/libopenapi/blob/main/datamodel/high/base/schema_proxy.go#L97
IMO should be safe to continue as is.